grc/scripts/freedesktop/org.gnuradio.grc.metainfo.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
Is this comment something to be concerned about?

Reply:
Yes. That's why I added it.